libs/sdk-backend-tiger/tests/integrated/backend.ts

```diff
@@ -66,5 +72,115 @@ export function testBackend(): IAnalyticalBackend {
 
 export function testWorkspace(): string {
     // UI SDK Reference workspace ID
-    return "4dc4e033e611421791adea58d34d958c";
+    const workspace: string = process.env.WORKSPACE_ID ?? "4aefdef78922461faeb08c12de183f81";
+    return workspace;
+}
+
+export function compareJsonDisregardTheOrder(jsonActual: any, jsonExpected: any) {
+    if (Object.prototype.toString.call(jsonActual) !== Object.prototype.toString.call(jsonExpected)) {
+        return false;
+    }
+
+    if (Array.isArray(jsonActual)) {
+        if (jsonActual.length !== jsonExpected.length) {
+            return false;
+        }
+
+        let i = jsonActual.length;
+        while (i--) {
+            let j = jsonExpected.length;
+            let found = false;
+
+            while (!found && j--) {
+                if (compareJsonDisregardTheOrder(jsonActual[i], jsonExpected[j])) found = true;
+            }
+
+            if (!found) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    if (Object.prototype.toString.call(jsonActual) === "[object Object]") {
+        for (const key in jsonActual) {
+            if (Object.prototype.hasOwnProperty.call(jsonActual, key)) {
+                if (!Object.prototype.hasOwnProperty.call(jsonExpected, key)) {
+                    return false;
+                }
+
+                if (!compareJsonDisregardTheOrder(jsonActual[key], jsonExpected[key])) {
+                    return false;
+                }
+            }
+        }
+
+        for (const key in jsonExpected) {
+            if (
+                Object.prototype.hasOwnProperty.call(jsonExpected, key) &&
+                !Object.prototype.hasOwnProperty.call(jsonActual, key)
+            ) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    return jsonActual === jsonExpected;
+}
+
+export async function sanitizeKeyWithNewValue(result: any, key: string, newValue: string) {
+    const fixtureContent = JSON.stringify(result);
+    const newResult = await JSON.parse(fixtureContent, (k, v) => (k === key ? newValue : v));
+    return newResult;
+}
+
+function getId(item: any) {
+    return item.attribute?.id || item.measure?.id || item.fact?.id || item.dataSet?.id || item.dateAttributes;
+}
+
+export async function sanitizeWorkspace(result: any, key: string, newValue: string) {
+    const newResult = await sanitizeKeyWithNewValue(result, key, newValue);
+    delete newResult.workspace;
+    return newResult;
+}
+
+export function sortToOder(o: any) {
```

Comment:
What about using JSON.parse() with replacer, to make it more universal?
- check if the value is array value, with objects with "id" property (and if so, replace it with the sorted one - similar to the mapping bellow)

Reply:
rewrite sortToOrder() to make it more universal, please check